src/autopas/options/IteratorBehavior.h

```diff
@@ -50,6 +73,14 @@ class IteratorBehavior : public Option<IteratorBehavior> {
    */
   constexpr IteratorBehavior(Value option) : _value(option) {}
 
+  /**
+   * Constructor from number value.
+   * This is useful when combining values and directly using the result as argument of type IteratorBehavior.
+   * This is necessary since e.g. owned & halo results in an object of Value_t instead of Value.
+   * @param option
+   */
+  constexpr IteratorBehavior(Value_t option) : _value(static_cast<IteratorBehavior::Value>(option)) {}
```

Comment:
Alternatively, one could define the `operator |` for this class:
https://godbolt.org/z/q4GjYndT4

Reply:
I actually considered this, however, for consistency all operators should then be defined and I just couldn't be bothered ;)